tests/transforms/test_classification.py

```diff
@@ -268,6 +269,50 @@ def test_classifier_model_path(
                 args=[])
         assert "does not exist" in str(e.value)
 
+    def test_invalid_model_registry_env(self, input_data):
+        # This should run without error
+        ArgSchemaParser(input_data=input_data,
+                        schema_type=InferenceInputSchema,
+                        args=[])
+        # Now to test when the model_registry_env field is invalid
+        invalid_model_registry_env = input_data.copy()
+        invalid_model_registry_env["model_registry_env"] = "prode"
+        with pytest.raises(ValidationError) as e:
```

Comment:
you can use the 'match' keyword with pytest.raises to provide a regex to match the exception. This would eliminate the need for the later assertion.

```
with pytest.raises(ValidationError, match="some_regex"):
```

Reply:
I purposely separated because using that form resulted in an over 80 char line (and I didn't like how the multi-line version looked)

Reply:
Haha makes sense to me, same result